components/CollectionCells/examples/CollectionCellsLayoutExample.m

```diff
@@ -156,7 +152,13 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView
   }
   if (model.circle) {
     cell.imageView.image =
-        [self imageWithSize:CGSizeMake(40, 40) color:HEXCOLOR(0x80CBC4) cornerRadius:20];
+    //80CBC4
```

Comment:
nit: space after // (I'd just remove it)

Reply:
Done